lib/katello/plugin.rb

```diff
@@ -171,4 +171,11 @@
                    :partial => 'smart_proxies/show/storage',
                    :onlyif => proc { |proxy| proxy.has_feature?(SmartProxy::PULP_FEATURE) || proxy.has_feature?(SmartProxy::PULP_NODE_FEATURE) }
   end
+
+  RemoteExecutionFeature.register(:katello_package_install, N_("Katello: Install Package"), :description => N_("Install package via Katello interface"), :provided_inputs => ['package'])
```

Comment:
You can. This mostly serves as a mapping to the other templates. This way we make sure, that the default teplates match the inputs with what Katello provides